src/models/data.cpp

```diff
@@ -12,6 +12,63 @@
 
 using namespace Data;
 
+void ItemCost::resize(std::size_t newSize, qint64 value)
```

Comment:
as I said before: all of these should be inline

Reply:
ah, that wasn't clear, will do